modules/homebrew/init.zsh

```diff
@@ -35,6 +35,12 @@ alias brews='brew search'
 alias brewu='brew upgrade'
 alias brewx='brew uninstall'
 
+# Homebrew Services
+alias bservl='brew services list'
+alias bservr='brew services restart'
+alias bservs='brew services start'
+alias bservx='brew services stop'
+
```

Comment:
While I like the idea, I am still on the fence regarding the choice of alias prefix.

I am not sure if we should keep using `brew<...>` or introduce yet another set `bserv<...>`.

Reply:
I'm happy to go with whatever, `brewsl`, `brewsr` etc. would work too.

Reply:
Sure @citizen428! `brews` is already taken by `brew search`, so something different might be preferred.

Maybe `brewSl`, `brewSr` etc. might be considered. But then again, we are using lower `s` and upper `S` to distingush between completely unrelated things.

Which is why I am not exactly sure (and on the fence) what might be a better convention. Let's see what others have to say.

Reply:
It seems no one has to say anything, so maybe let's just close this?

Reply:
I agree with @indrajitr that the prefix is a little odd, but I'd still be fine including this if you're still interested in the change.

Reply:
I personally don't use Prezto anymore, so it's up to you really. But if you prefer another prefix I can update the PR as a parting gift 😃

Reply:
IMO let's keep `brew` as the prefix for all Homebrew related commands. If someone types it so frequently that they want something shorter, they can add it to their personal setup.